test/unit/components/VExpansionPanel/VExpansionPanelContent.spec.js

```diff
@@ -78,6 +78,14 @@ test('VExpansionPanelContent.js', ({ mount, compileToFunctions }) => {
     expect(wrapper.html()).toMatchSnapshot()
   })
 
+  it('should warn a registrable warning when expansionPanel empty', () => {
```

Comment:
I don't think this test is needed, just changing the warning message should be fine.

Reply:
Even if it improves the code coverage?

Reply:
ok ! I did remove it! Thanks Kael